app/routes/user/email_notifications.server.ts

```diff
@@ -0,0 +1,208 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { tables } from '@architect/functions'
+import { mapTopicsToFormatAndNoticeType } from '~/lib/utils'
+import { getUser } from '~/routes/__auth/user.server'
+
+// db model
+export type EmailNotification = {
+  name: string
+  recipient: string
+  created: number
+  active: boolean
+  uuid?: string
+  topics: string[]
+}
+
+// view model
+export type EmailNotificationVM = EmailNotification & {
```

Comment:
Prefer interface over type intersections (see [TypeScript wiki](https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections)).

Reply:
Updated in latest